docs/beginner/tutorial2-surface/README.md

```diff
@@ -255,29 +277,11 @@ Don't use `block_on` inside of an async function if you plan to support WASM. Fu
 
 </div>
 
-If we try to build WASM now it will fail because `wasm-bindgen` doesn't support using async functions as `start` methods. You could switch to calling `run` manually in javascript, but for simplicity, we'll add the [wasm-bindgen-futures](https://docs.rs/wasm-bindgen-futures) crate to our WASM dependencies as that doesn't require us to change any code. Your dependencies should look something like this:
+<div class="note">
```

Comment:
Given that I need to annotate `run()` with `wasm-bindgen(start)`, this should probably just be part of the main text.

Reply:
I was a bit on sure on this one., as the `wasm-bindgen(start)` annotation is added in [Tutorial 1](https://github.com/sotrh/learn-wgpu/blob/a1f5995f498e5bd5cdf111155c16a681e5834396/docs/beginner/tutorial1-window/README.md?plain=1#L145). Would you prefer to drop the note tag instead?